src/button-fixed-footer-layout.tsx

```diff
@@ -41,8 +41,11 @@ const ButtonFixedFooterLayout: React.FC<Props> = ({
             containerBgColor={containerBgColor}
             footer={
                 <ResponsiveLayout>
-                    <Box paddingY={isMobile ? 16 : 0}>
-                        <ButtonLayout align={isMobile ? 'full-width' : desktopButtonAlign} link={link}>
+                    <Box paddingY={isTabletOrSmaller ? 16 : 0}>
```

Comment:
I don't know if this is the correct place, but, can we change paddings from 16px to 32px in tablet to avoid this weird thing?

![image](https://user-images.githubusercontent.com/6722153/129160369-5c10d5bf-2699-4252-9654-c5625501d9cf.png)

![image](https://user-images.githubusercontent.com/6722153/129160407-30e8836b-8896-4a4c-8928-e1bbd7ddab58.png)

![image](https://user-images.githubusercontent.com/6722153/129160469-d9662faa-22c2-4691-9669-33d353a50e51.png)

Reply:
only for tablet or for mobile+tablet?

Reply:
ok